db/v-2/tables/execution.xml

```diff
@@ -12,4 +12,15 @@
             </column>
         </addColumn>
     </changeSet>
-</databaseChangeLog>
\ No newline at end of file
+
+    <changeSet id="add-execCmd-and-batchSizeForAnalyzer-columns" author="kgevorkyan" context="dev or prod">
+        <addColumn tableName="execution">
+            <column name="exec_cmd" type="varchar(255)" defaultValue="NULL"/>
+        </addColumn>
+
+        <addColumn tableName="execution">
+            <column name="batch_size_for_analyzer" type="varchar(255)" defaultValue="NULL"/>
```

Comment:
Why varchar?

Reply:
We take this value from `input group` from frontend, which is `string`, and I don't see any reasons to cast it to the `int` for now, since for `save-cli` we also put `batchSize` into the `toml` file as `string`